src/python_pybind11/CMakeLists.txt

```diff
@@ -1,3 +1,19 @@
+# Detect if we are doing a standalone build of the bindings, using an external gz-math
+if(CMAKE_SOURCE_DIR STREQUAL CMAKE_CURRENT_SOURCE_DIR)
+  cmake_minimum_required(VERSION 3.16)
+  set(GZ_MATH_VER 8)
```

Comment:
I'm afraid about keeping this number in sync with bumps. As a workaround we can put a comment in the main `CMakeLists.txt` pointing to here. In the mid-term, it would be nice to integrate our CMake code to grab the version/name from git directly, probably using something like https://github.com/LecrisUT/CMakeExtraUtils/blob/main/cmake/DynamicVersion.md

Reply:
I am not a big fan of using git for version information as fails for tarballs. Something a bit ugly but effective could be to grep the major version from the parent folder CMakeLists.txt .

Reply:
prototype for extracting version numbers from package.xml: https://github.com/gazebosim/gz-math/pull/639

Reply:
I've also opened https://github.com/gazebosim/gz-cmake/pull/456 adding the python script from https://github.com/gazebosim/gz-math/pull/639 and a cmake helper function. If you don't mind approving this PR for now so we can start to fix CI, I will follow-up by using the gz-cmake helper once it has been merged and released